vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -613,9 +615,21 @@ def query_vsan_metrics(self, starting_time):
                 self.log.debug("Skipping vsan metrics for cluster %s because it is not a vsan cluster", cluster.name)
         if not cluster_nested_elts:
             self.log.debug("There are no vsan clusters to collect metrics from, skipping vsan collection")
-            return [], []
+            return [], [], []
         return self.api.get_vsan_metrics(cluster_nested_elts, entity_ref_ids, id_to_tags, starting_time)
 
+    def send_resource_to_redapl(self, clusters_data):
+        input_data = {}
+        for cluster in clusters_data:
+            for i, v in cluster.items():
+                if i not in input_data:
+                    input_data[i] = ''
+                input_data[i] += str(v) + ','
+        for i, v in input_data.items():
+            # we want to remove the last comma to avoid adding an empty '' resource
+            self.log.debug("Sending resource to redapl: %s, %s", i, v.rstrip(','))
+            self.set_metadata(i, v.rstrip(','))
```

Comment:
What do `i` and `v` represent? Could we rename these?

Reply:
An example would be `size_used:3`, so I will change `i` to `cluster_tag` and `v` to `cluster_tag_value`.